### PR TITLE
String pin numbers should be converted to type number when converting to tsx

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/tscircuit/easyeda-converter/issues/80

### CODE REVIEW

File: lib/convert-to-typescript-component/index.tsx [hunk 1]
```diff
@@ -32,33 +32,56 @@ export const convertBetterEasyToTsx = async ({
...
+      // const newLabel =
+      //   originalPinNumber === pin.label
+      //     ? [pin.label]
+      //     : [originalPinNumber, pin.label]
+
+      return {
+        ...pin,
+        pinNumber: newPinNumber,
+        label: pin.label,
+      }
```
Comment (andrii-balitskyi) [inline lib/convert-to-typescript-component/index.tsx:58]: Currently it looks like this, is it correct? 
image

Or should it look like this?
image

Comment (andrii-balitskyi) [reply lib/convert-to-typescript-component/index.tsx:58]: There's also duplicate `SH` label? Should we check for duplicate labels and add some unique characters?

Comment (seveibar) [reply lib/convert-to-typescript-component/index.tsx:58]: Nope duplicates are ok!!!

**Review by seveibar — APPROVED** [review]
Nice!

File: lib/convert-to-typescript-component/index.tsx (hunk 1 above)
Comment (andrii-balitskyi) [reply lib/convert-to-typescript-component/index.tsx:58]: @seveibar my bad, should've marked the PR draft. There's some ghost code as I wanted to confirm the structure. 
So, the first screenshot looks right?

**Pull request merged**